libfreerdp/core/nla.c

```diff
@@ -115,8 +115,26 @@ static void nla_identity_free(SEC_WINNT_AUTH_IDENTITY* identity);
 #define ber_sizeof_sequence_octet_string(length) ber_sizeof_contextual_tag(ber_sizeof_octet_string(length)) + ber_sizeof_octet_string(length)
 #define ber_write_sequence_octet_string(stream, context, value, length) ber_write_contextual_tag(stream, context, ber_sizeof_octet_string(length), TRUE) + ber_write_octet_string(stream, value, length)
 
-static const CHAR ClientServerHashMagic[] = "CredSSP Client-To-Server Binding Hash\0";
-static const CHAR ServerClientHashMagic[] = "CredSSP Server-To-Client Binding Hash\0";
+/* CredSSP Client-To-Server Binding Hash\0 */
+static const BYTE ClientServerHashMagic[] =
```

Comment:
Does VS complain when initializing `BYTE` (aka `unsigned char`) simply as string?

Reply:
VS 2017 has no complaint about it - but then there are other compilers out there as we know. Also this way we ensure that there is definitely just one '\0' at the end of the string - compilers are supposed to add one at the end when defining strings this way ... supposed to.

Reply:
@mfleisz  ok, keeping the magic :)